chromium_src/chrome/browser/net/system_network_context_manager.cc

```diff
@@ -4,4 +4,10 @@
  * You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 #include "brave/services/network/public/cpp/system_request_handler.h"
+
+#define BRAVE_SYSTEM_NETWORK_CONTEXT_MANAGER_CONFIGURE_DEFAULT_NETWORK_CONTEXT_PARAMS \
+  quic_user_agent_id = "";
```

Comment:
I think we can avoid this patch by doing an override for `network_session_configurator::ParseCommandLineAndFieldTrials`